### DISCUSSION TITLE
Updating label text to reflect CTkSlider selected values

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#227] [Q&A] [answered]
Hi. I have just, this week, learned tkinter and subsequently discovered customtkinter. I have developed my first application, which I am porting to customekinter. The posrted app now includes several new sliders (CTkSlider). Unlike the bog standard tkinter sliders, there is no value display. To compensate for this I was hoping to define a CTkLabel to the side of each slider. The problem i am having is finding a way to update the label text, when the slider is rmoved.

Open to any suggestions on this one - maybe my approach is wrong?

Thanks,

Clive

**Comment by avalon60** [comment, marked as answer]
Just spotted that there is a method described at the bottom of the wiki - will give it a whirl!
So the method is described on the wiki:
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkLabel

I used CTkLabel.configure(text=new_text)

**Reply by felipetesc** [reply]
Hello [avalon](https://github.com/avalon60) ! Is this solved ? If positive can you close it ? And perhaps share what you have done to update the label text value when the slider is moved ?

**Reply by avalon60** [reply]
Hi. Put my solution in, but not sure how to close it?
